src/components/main/mainContainer.jsx

```diff
@@ -1,16 +1,21 @@
-import React from 'react';
+import React, { useState } from 'react';
 import { Switch, Route, Redirect } from 'react-router-dom';
-import { Container, Typography } from '@material-ui/core';
+import { Container, Typography } from '@mui/material';
 import SideBar from '../sidebar/sideBar';
+import ContentHeader from '../contentHeader/contentHeader';
 import './mainContainer.scss';
 
 const MainContainer = () => {
+  const [routeIsEmployee, setRouteIsEmployee] = useState(false);
```

Comment:
Hm, do we really need this? I mean, what if we will handle which page is opened, by using `location`(history)?
E.g if `location.path` is/or contain `/department` than header will be `Departments` and so on